utils/titleize.js

```diff
@@ -0,0 +1,3 @@
+export default function titleize(input) {
```

Comment:
we use to use `inflected` in Fino for theses transforms but I think it's OK for now 👍